When creating an image sequence,

JacksonRG opened the pull request:
Create image sequences that match the current project's fps.

Commit:
When creating an image sequence,
get the project's FPS and apply that to the sequence

jonoomph commented:
LGTM

Commit:
Don't apply project fps to animated titles

Blender animated titles specify their FPS
Other Image sequences default to the project fps

jonoomph commented on src/windows/models/files_model.py:
```diff
@@ -324,9 +324,17 @@ def add_files(self, files, image_seq_details=None, quiet=False, prevent_image_se
-                    project_fps_den = get_app().project.get("fps").get("den", 1)
+                    if seq_info and "fps" in seq_info:
+                        # Blender Titles specify their fps in seq_info
+                        log.debug("Image Sequence using FPS specified in seq_info")
```
I would love to see the FPS printed in the log.debug messages

jonoomph commented:
Small nitpick, but otherwise LGTM

Commit:
Show FPS in log statment

Also removed "project" from the fps variable name, since
it might not be the FPS of the project.

Commit:
Default animated titles to 25 fps

- Specify 25 fps for animated titles, since that's the rate they're designed for.
